test/common/integreatly_stages.go

```diff
@@ -0,0 +1,34 @@
+package common
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+)
+
+func TestIntegreatlyStagesStatus(t *testing.T, ctx *TestingContext) {
+
+	//get RHMI
+	rhmi, err := getRHMI(ctx.Client)
+	if err != nil {
+		t.Fatalf("error getting RHMI CR: %v", err)
+	}
+
+	//iterate stages and check their status
+	// This does not check the pro
+	for _, stage := range rhmi.Status.Stages {
+		err := checkStatus(stage)
+		if err != nil {
+			t.Logf("Error: Product %v not completed. It's current status is %v", stage.Name, err)
```

Comment:
Should this be `t.Fatalf` instead of a log?

Reply:
yeah. it should. Will push an update